.github/workflows/publish-from-package.yml

```diff
@@ -5,6 +5,7 @@ on: [workflow_dispatch]
 jobs:
   release:
     name: Release
+    if: "!(github.actor == 'github-actions[bot]' and contains(github.event.commits[0].message, 'chore(release): publish'))"
```

Comment:
I'm not sure we necessarily need to make any changes to this workflow. This one needs to be triggered manually on `master` once Lerna's `chore: publish` commit lands on `master`. There's no commit associated with the run of this workflow.